ci-operator/config/openshift-knative/eventing-istio/OWNERS

```diff
@@ -0,0 +1,23 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# Fetched from https://github.com/openshift-knative/eventing root OWNERS
+# If the repo had OWNERS_ALIASES then the aliases were expanded
+# Logins who are not members of 'openshift' organization were filtered out
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- alanfx
+- aliok
+- creydr
+- jcrossley3
```

Comment:
I think we can unfortunately remove Jim here

Reply:
I copied it from eventing directory